testfiles/.editorconfig

```diff
@@ -21,3 +21,7 @@ max_line_length = 2
 [spaces-with-tab.c]
 indent_style = space
 indent_size = unset
+
+[multiple-wrong-indentations.txt]
+indent_style = space
+indent_size = 10
```

Comment:
As the "testfile"  has been removed in [`42f72b3` (#360)](https://github.com/editorconfig-checker/editorconfig-checker/pull/360/commits/42f72b341487979acfb5e6a36f9cbdb3d712f843), those lines should also be removed.

Reply:
oh, good point. Thanks, I totally forgot that one. Now it is removed.